### PR TITLE
Classloader hack

### PR DESCRIPTION
**Pull request by mkristian**

### CODE REVIEW

File: lib/fast-rsa-engine.rb [hunk 1]
```diff
@@ -5,6 +5,15 @@
 
   # keep the default name space clean and use tap
   tap do
+    begin
+      old = java.lang.Thread.currentThread.context_class_loader
```
Comment (rtyler) [inline lib/fast-rsa-engine.rb]: This code needs a *lot* more commentary and documentation around it IMHO

File: fast-rsa-engine.gemspec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |s|
   s.name = 'fast-rsa-engine'
-  s.version = '0.3.0'
+  s.version = '0.3.1'
```
Comment (rtyler) [inline fast-rsa-engine.gemspec]: Do you think this change is small enough to be acceptable in a patch-level version change?

Comment (mkristian) [inline fast-rsa-engine.gemspec]: for me it is a bug fix but one can argue it is new feature :)

**Comment by mkristian** [comment]
closing in favor of #21

**Pull request closed without merging**